lib/Bio/EnsEMBL/DataCheck/Checks/DisplaybleGenes.pm

```diff
@@ -0,0 +1,69 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::DisplaybleGenes;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Utils qw/sql_count/;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'DisplaybleGenes',
+  DESCRIPTION    => 'Check that genes are displayable and '
+    . 'have web_data attached to their analysis.',
+  GROUPS         => ['core_handover'],
+  DATACHECK_TYPE => 'advisory',
+  DB_TYPES       => ['core', 'otherfeatures', 'rnaseq', 'cdna'],
+  TABLES         => ['gene', 'analysis', 'analysis_description']
+};
+
+sub tests {
+  my ($self) = @_;
+
+  sub skip_tests {
```

Comment:
I don't think this is required; we should pretty much always have analysis_description rows. So we're doing a redundant count, since we will almost never need to skip.
In any case, this sub should not be within the `tests` sub, it should just be alongside it.

Reply:
`skip_tests` is no longer within `tests`, but my main point here was that it shouldn't be there at all...